pwiz_tools/Skyline/Skyline.csproj

```diff
@@ -1754,9 +1757,9 @@
     <Compile Include="Model\ModificationMatcher.cs" />
     <Compile Include="Model\StandardType.cs" />
     <Compile Include="Model\Themes\ColorScheme.cs" />
-    <Compile Include="Model\Tools\ToolDescription.cs" />
-    <Compile Include="Model\Tools\ToolInstaller.cs" />
-    <Compile Include="Model\Tools\ToolMacros.cs" />
+    <Compile Include="ToolsUI\ToolDescription.cs" />

```

Comment:
You can't move all of these to ToolsUI, because we need to support tool installation from the command-line so that a 3rd party tool can install a tool on the Skyline Tools menu.